sky/optimizer.py

```diff
@@ -208,33 +285,36 @@ def _get_egress_info(
                 # A task_lib.Task may have no inputs specified.
                 return None, None, 0
             src_cloud = node.get_inputs_cloud()
-            nbytes = node.get_estimated_inputs_size_gigabytes()
+            n_gigabytes = node.get_estimated_inputs_size_gigabytes()
```

Comment:
This should be deprecated as well? Actually, we should add edge info (and set data to 0) when we are adding the dummy source/sink node?

Reply:
adfter that, we could remove the `node` argument as well.

Reply:
Actually, after a second look, this code path is handling the case for reading from the existing bucket (e.g. the original task yaml exists some bucket mount to some specified cloud). lets add a todo to handle such case.